gr-azure-software-radio/NOTICE

```diff
@@ -0,0 +1 @@
+TODO: Update with any required Third-party open source declarations
```

Comment:
This looks like boilerplate. Thoughts on pointing to GNU Radio here?

Reply:
I don't think we need any declarations